.test.mk

```diff
@@ -172,7 +172,7 @@ test-osx-release: prebuild-osx build run-luajit-test pretest-osx run-test
 test-osx-release-arm64: CMAKE_PARAMS = -DCMAKE_BUILD_TYPE=RelWithDebInfo \
                                        -DENABLE_WERROR=ON \
                                        -DTEST_BUILD=ON
-test-osx-release-arm64: prebuild-osx build
+test-osx-release-arm64: prebuild-osx build run-luajit-test pretest-osx run-test
```

Comment:
Please remove this target at all. You have enabled testing, so you can use `test-osx-release` target. The same for debug and static builds. Also, please fix all macOS workflows: delete this logic https://github.com/tarantool/tarantool/blob/9549310d15dd16c6fd49cc658bdf9fffb82ba63c/.github/workflows/osx_debug.yml#L61-L62.

Reply:
@ylobankov, thanks for your review! I've fixed all spots you've mentioned above.

```diff
diff --git a/.github/workflows/osx_debug.yml b/.github/workflows/osx_debug.yml
index 515dc51ec..a53e0c678 100644
--- a/.github/workflows/osx_debug.yml
+++ b/.github/workflows/osx_debug.yml
@@ -58,9 +58,7 @@ jobs:
       - name: Install deps
         uses: ./.github/actions/install-deps-osx
       - name: test
-        env:
-          SUFFIX: ${{ matrix.arch == 'aarch64' && '-arm64' || '' }}
-        run: make -f .test.mk test-osx-debug${SUFFIX}
+        run: make -f .test.mk test-osx-debug
       - name: Send VK Teams message on failure
         if: failure()
         uses: ./.github/actions/report-job-status
diff --git a/.github/workflows/osx_release.yml b/.github/workflows/osx_release.yml
index 425bc96b1..fb18179ab 100644
--- a/.github/workflows/osx_release.yml
+++ b/.github/workflows/osx_release.yml
@@ -58,9 +58,7 @@ jobs:
       - name: Install deps
         uses: ./.github/actions/install-deps-osx
       - name: test
-        env:
-          SUFFIX: ${{ matrix.arch == 'aarch64' && '-arm64' || '' }}
-        run: make -f .test.mk test-osx-release${SUFFIX}
+        run: make -f .test.mk test-osx-release
       - name: Send VK Teams message on failure
         if: failure()
         uses: ./.github/actions/report-job-status
diff --git a/.github/workflows/osx_release_lto.yml b/.github/workflows/osx_release_lto.yml
index 0192e1c2b..f81139316 100644
--- a/.github/workflows/osx_release_lto.yml
+++ b/.github/workflows/osx_release_lto.yml
@@ -60,8 +60,7 @@ jobs:
       - name: test
         env:
           CMAKE_EXTRA_PARAMS: -DENABLE_LTO=ON
-          SUFFIX: ${{ matrix.arch == 'aarch64' && '-arm64' || '' }}
-        run: make -f .test.mk test-osx-release${SUFFIX}
+        run: make -f .test.mk test-osx-release
       - name: Send VK Teams message on failure
         if: failure()
         uses: ./.github/actions/report-job-status
diff --git a/.github/workflows/osx_static_cmake.yml b/.github/workflows/osx_static_cmake.yml
index 6208f839c..a34c158e8 100644
--- a/.github/workflows/osx_static_cmake.yml
+++ b/.github/workflows/osx_static_cmake.yml
@@ -60,8 +60,7 @@ jobs:
         uses: ./.github/actions/install-deps-osx
       - name: test
         env:
-          SUFFIX: ${{ matrix.arch == 'aarch64' && '-arm64' || '' }}
-        run: make -f .test.mk test-osx-static-cmake${SUFFIX}
+        run: make -f .test.mk test-osx-static-cmake
       - name: Send VK Teams message on failure
         if: failure()
         uses: ./.github/actions/report-job-status
diff --git a/.test.mk b/.test.mk
index d2f80c430..5b9e884cf 100644
--- a/.test.mk
+++ b/.test.mk
@@ -164,16 +164,6 @@ test-osx-release: CMAKE_PARAMS = -DCMAKE_BUILD_TYPE=RelWithDebInfo \
                                  -DTEST_BUILD=ON
 test-osx-release: prebuild-osx build run-luajit-test pretest-osx run-test

-# FIXME: Temporary target without tests. Use 'test-release-osx' target instead
-# when all M1 issues are resolved:
-#   LuaJIT tests - https://github.com/tarantool/tarantool/issues/4819
-#   Tarantool tests - https://github.com/tarantool/tarantool/issues/6068
-.PHONY: test-osx-release-arm64
-test-osx-release-arm64: CMAKE_PARAMS = -DCMAKE_BUILD_TYPE=RelWithDebInfo \
-                                       -DENABLE_WERROR=ON \
-                                       -DTEST_BUILD=ON
-test-osx-release-arm64: prebuild-osx build run-luajit-test pretest-osx run-test
-
 # Debug build

 .PHONY: test-osx-debug
@@ -181,15 +171,6 @@ test-osx-debug: CMAKE_PARAMS = -DCMAKE_BUILD_TYPE=Debug \
                                -DTEST_BUILD=ON
 test-osx-debug: prebuild-osx build run-luajit-test pretest-osx run-test

-# FIXME: Temporary target without tests. Use 'test-debug-osx' target instead
-# when all M1 issues are resolved:
-#   LuaJIT tests - https://github.com/tarantool/tarantool/issues/4819
-#   Tarantool tests - https://github.com/tarantool/tarantool/issues/6068
-.PHONY: test-osx-debug-arm64
-test-osx-debug-arm64: CMAKE_PARAMS = -DCMAKE_BUILD_TYPE=Debug \
-                                     -DTEST_BUILD=ON
-test-osx-debug-arm64: prebuild-osx build run-luajit-test pretest-osx run-test
-
 # Static build

 .PHONY: test-osx-static-cmake
@@ -201,19 +182,6 @@ test-osx-static-cmake: LUAJIT_TEST_BUILD_DIR = ${STATIC_BIN_DIR}
 test-osx-static-cmake: TEST_RUN_PARAMS = --builddir ${PWD}/${STATIC_BIN_DIR}
 test-osx-static-cmake: prebuild-osx build run-luajit-test pretest-osx run-test

-# FIXME: Temporary target without tests. Use 'test-osx-static-cmake' target instead
-# when all M1 issues are resolved:
-#   LuaJIT tests - https://github.com/tarantool/tarantool/issues/4819
-#   Tarantool tests - https://github.com/tarantool/tarantool/issues/6068
-.PHONY: test-osx-static-cmake-arm64
-test-osx-static-cmake-arm64: SRC_DIR = ${STATIC_DIR}
-test-osx-static-cmake-arm64: BUILD_DIR = ${STATIC_DIR}
-test-osx-static-cmake-arm64: CMAKE_PARAMS = -DCMAKE_TARANTOOL_ARGS="-DCMAKE_BUILD_TYPE=RelWithDebInfo;-DENABLE_WERROR=ON;-DTEST_BUILD=ON"
-test-osx-static-cmake-arm64: CTEST = true
-test-osx-static-cmake-arm64: LUAJIT_TEST_BUILD_DIR = ${STATIC_BIN_DIR}
-test-osx-static-cmake-arm64: TEST_RUN_PARAMS = --builddir ${PWD}/${STATIC_BIN_DIR}
-test-osx-static-cmake-arm64: prebuild-osx build
-
 ##############################
 # FreeBSD                    #
 ##############################
```